src/Database/Schema/PostgresSchema.php

```diff
@@ -334,7 +334,10 @@ public function columnSql(Table $table, $name)
             'binary' => ' BYTEA',
             'float' => ' FLOAT',
             'decimal' => ' DECIMAL',
+            'tinytext' => ' TEXT',
```

Comment:
I would this so. `medium` text is just `TEXT` and `LONG TEXT` is bigger than `TEXT`.